packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -70,18 +74,23 @@ class="fixed inset-0 z-40 flex items-center min-h-screen p-4 overflow-y-auto tra
                 x-on:keydown.window.escape="isOpen = false"
             @endif
             x-transition:enter="ease duration-300"
+            @if (! $isFullscreen)
             x-transition:enter-start="translate-y-8"
             x-transition:enter-end="translate-y-0"
+            @endif
             x-transition:leave="ease duration-300"
+            @if (! $isFullscreen)
             x-transition:leave-start="translate-y-0"
             x-transition:leave-end="translate-y-8"
+            @endif
```

Comment:
Maybe move into a single condition, even though the phases are out of order in the code then?

Also fix indentation please. 🙂

Reply:
Fixed in latest commit.